### ISSUE TITLE
ReferenceError: args is not defined

### ISSUE THREAD

**Issue by tume** [rcsole/gulp-cordova#11]
When trying to run any 'run' commands I'm getting this:
```
ReferenceError: args is not defined
  at cordovaWrapper (/Users/tume/Projects/nano/nano_frontend/node_modules/gulp-cordova/wrapper.js:58:25)
  at execute (/Users/tume/Projects/nano/nano_frontend/node_modules/gulp-cordova/index.js:77:5)
  at /Users/tume/Projects/nano/nano_frontend/node_modules/gulp-cordova/index.js:50:7
```

**Comment by rcsole** [comment]
Hi @tume could you provide an example? Thanks!

**Comment by tlkiong** [comment]
Waiting to have this fixed in @Matzu89 's pull request.
For now, its a manual removal of lines 58 to 61 in wrapper.js file

**Comment by rcsole** [comment]
Closed by #14
